scripts/privacyidea

```diff
@@ -988,7 +988,7 @@ def create_arguments():
                           "of length",
                           metavar="NUMBER",
                           type=int,
-                          default=0)
+                          default=6)
```

Comment:
The --yubikeyprefixrandom works for all yubikey token types. So if you set the default to 6 in the parameters, it will output the prefix also in front of the oath tokens. 
Plus: I also (and only) know users till now, that use the yubikey-type against /validate/check without prefix.
This is why we need to keep the default=0.
Thanks a lot
Cornelius